packages/lib-classifier/src/plugins/tasks/DrawingTask/models/DrawingTask.js

```diff
@@ -15,6 +15,8 @@ export const Drawing = types.model('Drawing', {
   activeToolIndex: types.optional(types.number, 0),
   annotation: types.safeReference(DrawingAnnotation),
   help: types.optional(types.string, ''),
+  hidePreviousMarks: types.optional(types.boolean, false),
+  hidingIndex: types.maybeNull(types.number),
```

Comment:
This works with the transcription task model for now (as its a composite with the drawing task model), but  once the 3 state version is implemented for it, this is something that would likely need to differentiate?

Reply:
Yep. This will have to change a bit for the three state version. I'll probably have to turn this boolean into an enum, or something similar.

Reply:
Cool, just wanted to check in that you had an idea of a plan of it. Thanks.